### PR TITLE
fix: Change enum value casing

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Change enum value casing as suggested by `clippy`

Partially addresses #225

### CODE REVIEW

File: src-vue/src/components/PullRequestsSelector.vue [hunk 1]
```diff
@@ -71,10 +71,10 @@ export default defineComponent({
         onChange(e: Object) {
             const openedCollapseNames = Object.values(e);
             if (openedCollapseNames.includes('1') && this.pull_requests_launcher.length === 0) {
-                this.getPullRequests('LAUNCHER');
+                this.getPullRequests('Launcher');
```
Comment (Alystrasz) [inline src-vue/src/components/PullRequestsSelector.vue:74]: Actually, why is `PullRequestType` not an enumeration?
I'd expect something like this:
```javascript
this.getPullRequests(PullRequestType.Launcher);
```

File: src-vue/src/components/PullRequestsSelector.vue [hunk 2]
```diff
@@ -71,10 +71,10 @@ export default defineComponent({
...
             }
             if (openedCollapseNames.includes('2') && this.pull_requests_mods.length === 0) {
-                this.getPullRequests('MODS');
+                this.getPullRequests('Mods');
```
Comment (Alystrasz) [inline src-vue/src/components/PullRequestsSelector.vue:77]: Same remark here

**Review by Alystrasz — CHANGES_REQUESTED** [review]
I might have omitted an issue while merging PR-install feature :)
Otherwise, code looks good and runs fine on Ubuntu.

File: src-vue/src/components/PullRequestsSelector.vue (hunk 1 above)
Comment (GeckoEidechse) [reply src-vue/src/components/PullRequestsSelector.vue:74]: Cause for some reason `ts-rs` exports it as a set of strings and I don't have anyway of changing that unfortunately ._.

https://github.com/R2NorthstarTools/FlightCore/pull/255/files#diff-68931ccb7f8858bb1c8ae0d7109db6ab91c0531c6233d5e036a1e5faf4277bebR3

See also https://github.com/Aleph-Alpha/ts-rs/issues/23

**Review by Alystrasz — APPROVED** [review]

**Pull request merged**